smallest_multiple.js

```diff
@@ -6,9 +6,19 @@
  * @return { Number }         Lowest Positive Number that is evenly divisible by all numbers
  *                            between 1 and `ceiling`
  */
-module.exports = function( ceiling ) {
+module.exports = function (ceiling) {
   // do work here
+  let x = ceiling;
+  let numberArray = [];
 
-
-  return 0;
+  for (let i = 1; i <= ceiling; i++) {
+    numberArray.push(i);
+  }
+  for (let j = 0; j < numberArray.length; j++) {
+    if (x % j !== 0) {
+      x += ceiling;
```

Comment:
I think by incrementing by `ceiling` there could be a chance that you might skip over the smallest multiple. I haven't tested this out but that's my guy reaction to seeing this code.

Reply:
Taking this back, I just realized that the ceiling is going to be a multiple of the answer so this works fine.